_posts/2018-10-10-Generating-Catalan-Orderings.md

```diff
@@ -89,7 +89,7 @@ As far as an implementation for this problem goes, we basically have two options
 
 We see these choices in a lot of combinatorial situations. Carrying out (1) is much easier and often requires
 much less work, since we don't actually have to enumerate over everything (+ even less so when there is a closed
-form solution). We can see it as a dynamic programming problem in which we build a table of the Catalan numbers
+form solution). We can see it as a dynamic programming, problem in which we build a table of the Catalan numbers
```

Comment:
I'd like this comma stricken from the record.

Reply:
Let the defense register skepticism with the court on this ruling. "Skepticism registered". B3T